### ISSUE TITLE
Setup wizard 

### ISSUE THREAD

**Issue by cineafx** [icecreamdatabase/dashboard#2]
Have it that a streamer is guided through every settings during setup.

**Comment by TranRed** [comment]
I personally think a setup wizard should only guide through the important step on not each and every single setting.

Should roughly be:

- Voice (recommended EN: Brian, recommended BR: ricardo)
- [optional I guess]Queue mode (yes/no). maybe with sound example 
- [optional I guess]Conversation mode (yes/no). maybe with sound example
- Setting up and linking your reward
- adding browser source to OBS
- free characters vs. pre-paid and possibility to buy

Everything else should be good enough as default if you just want to get started.

@cineafx Do you have any stats about what settings people actually changed? All the settings that are mostly left on default could be excluded from the setup wizard imho

**Comment by cineafx** [comment]
Yes "every" is a bit of an overstatement. Those for example would be things to do in a setup wizard.

I got stats but I don't have them aggregated currently. I would have to do that manually once.
Voice, volume and queue are probably the most used once. But volume is not something we need in the setup wizard.
